services/client/util.go

```diff
@@ -194,3 +211,22 @@ func (su shardByIDAscending) Less(i, j int) bool {
 func (su shardByIDAscending) Swap(i, j int) {
 	su[i], su[j] = su[j], su[i]
 }
+
+func adKey(sid services.ServiceID, id string) string {
+	return fmt.Sprintf("[%s][%s]", serviceKey(sid), id)
```

Comment:
yeah we have to migrate everything since we are gonna use a different data store, but should be simple to use the existing m3admin code with new kv store